File: src/main/java/com/example/day06/leetcode_1/Solution.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dic = dict()
        for i,num in enumerate(nums):
            if target - num in dic:
                return [dic[target - num], i]
            dic[num] = i
        return []

File: src/main/java/com/example/day06/leetcode_1/Solution.java
package com.example.day6.leetcode_1;

import java.util.HashMap;
import java.util.Map;

class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer,Integer> map = new HashMap<>();
        for(int i = 0;i < nums.length;i++){
            if(map.get(target - nums[i]) != null){
                return new int[]{i,map.get(target- nums[i])};
            }
            map.put(nums[i],i);
        }
        return new int[]{0,0};
    }
}